### ISSUE TITLE
Incomplete cost calculations for trolls (and maby for dwarfs too).

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#89]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/15_

----

```
What steps will reproduce the problem?
1. Create a troll
2. Spend all your money on resources.
3. Check your balance.

What is the expected output? What do you see instead?
Expected:
1a. Troll buying a Harley-Davidson, costs 18.000 nuyen.
1b. Troll buying all his/her equipment, total cost raised by 50%

Instead:
1a. Troll buying a Harley-Davidson, costs 12.000 nuyen.
1b. Troll buying all his/her equipment, total cost not raised.

What version of the product are you using? On what operating system?
Version 0.0.5.139

Please provide any additional information below.
SR5 94:
When a character factors in racial modifiers for gear costs (trolls have a 50% 
gear and Lifestyle cost increase, dwarfs have a 10% increas in gear costs), the 
player may choose to increase the cost for each item picked up at that time to 
help them track how much they've spent and how much nuyen ther have left. 
Alternately, they may choose to purchaise all the gear at the base cost, 
arriving at a subtotal, and then add in the appropriate cost modifier (10 or 50 
percent) to get the final total.

SR5 463:
Harley-Davidson Scorpion Cost: 12.000 nuyen.

Note: The costs of lifestyle for a troll is calculated correct.



```

Original issue reported on code.google.com by `a.steenv...@vista-online.nl` on 29 Sep 2014 at 7:47

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
[deleted comment]
```

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
Oh, I forgot. OS: Windos 7 Ultimate 64 bit, SP1

And one more typo: the costs of lifestyle for a troll is *NOT* calculated 
correct.
```

Original comment by `a.steenv...@vista-online.nl` on 29 Sep 2014 at 8:13

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
This isn't really correct. It's been errata'd. Cost for troll and ork are ONLY 
present in LIFESTYLE costs. Not purchasing gear, at all, ever.
```

Original comment by `Brad.J.L...@gmail.com` on 30 Dec 2014 at 8:21

**Comment by chummer5a** [comment]
Metatypes do not appear to be counted in Lifestyle costs. Other issue is working as intended though.

**Comment by chummer5a** [comment]
Resolved in 5.159.
